docs/reconstruction.md

```diff
@@ -12,7 +12,8 @@ A number of reconstruction techniques are available in Tomviz:
  * Algebraic Reconstruction Technique (ART)
  * Simultaneous Iterative Reconstruction Technique (SIRT)
  * TV minimization method
- 
+ * Tomopy
```

Comment:
Tompy => TomPy

Reply:
Ah, you found one :slightly_smiling_face: